cmd/util/make.go

```diff
@@ -295,6 +295,9 @@ func makeECHKey(template ECHConfigTemplate, inCertPath, outPath, outKeyPath stri
 	if len(clientFacingCert.DNSNames) == 0 {
 		fatalIfErr(errors.New("input certificate does not specify a DNS name"), keyGenErrMsg)
 	}
+	if name := clientFacingCert.DNSNames[0]; !isDomainName(name) {
```

Comment:
This is probably fine for now, but I wonder how we should note the requirement that the first entry in the SAN list be a FQDN. Maybe a comment [in the Config struct](https://github.com/xvzcf/tls-interop-runner/blob/eba8f0baec79be5aa760403448e92aa7f2f925f4/cmd/util/common.go#L95), saying that at least one Hostname MUST be a FQDN?

Reply:
Done, though this makes me realize that perhaps we should do this differently. Perhaps we should specify the public name via the `-host` flag rather than using the hostname of a certificate?

Reply:
Yeah,  that does seem much less fragile. Want to do that in a separate PR?

Reply:
I think it makes sense to do it here. Go ahead and unnapprove and I'll ping you when it's ready.

Reply:
Done.